app/views/organizations/staff/adoption_application_reviews/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render DashboardPageComponent.new(crumb: :dashboard_applications, crumb_options: [@application]) do |c| %>
+  <% c.with_header_title do %>
```

Comment:
Can we update this header to be 'Form Answers'
And then underneath the breadcrumbs, we can show the person's name and email
<img width="283" alt="image" src="https://github.com/user-attachments/assets/12c2be14-1886-4d46-942b-d41c2e0e640b">

Can we update this to

Reply:
Is this better?

![Alta-application-review-answers](https://github.com/user-attachments/assets/21bd5568-1c91-4fe9-a8e7-ad36f4108e1a)

Reply:
@sarvaiyanidhi can we put the email below the user's name and make it a hyperlink?

Joe Brando
[joe@brando.com](joe@brando.com)

Reply:
Done

![Alta-application-review-answers](https://github.com/user-attachments/assets/b1863d14-2564-468e-bbb6-7d88ce2a2492)